polaris-icons/svgo.config.js

```diff
@@ -33,18 +25,58 @@ module.exports = {
           mergePaths: false,
 
           convertPathData: {
+            applyTransforms: true,
+            applyTransformsStroked: true,
+            makeArcs: {
+              threshold: 2.5,
+              tolerence: 0.5,
+            },
+            straightCurves: true,
+            lineShorthands: true,
+            convertToZ: true,
+            curveSmoothShorthands: true,
+            floatPrecision: 3,
+            transformPrecision: 5,
+            removeUseless: true,
+            collapseRepeated: true,
             /**
              * Mixing absolute and relative path commands can lead to rendering
              * issues on some platforms. This disables converting some path data to
              * absolute if it is shorter, keeping all path data relative. Using
-             * relative paths means that data points are relative  to the current
+             * relative paths means that data points are relative to the current
              * point at the start of the path command, which does not greatly
              * increase the quantity of path data.
              */
             utilizeAbsolute: false,
+            leadingZero: true,
+            negativeExtraSpace: true,
+            noSpaceAfterFlags: false,
+            forceAbsolutePath: false,
           },
```

Comment:
Overriding this object stopped `svgo` from properly optimising the paths. You can validate this by removing these values and running `yarn test` it will now fail as the SVG paths are not being properly optimised. Seems like an issue with `svgo` but fixing it here for us first.